README.md

```diff
@@ -35,7 +35,7 @@ An extremely fast Python linter and code formatter, written in Rust.
 - 📏 Over [800 built-in rules](https://docs.astral.sh/ruff/rules/), with native re-implementations
     of popular Flake8 plugins, like flake8-bugbear
 - ⌨️ First-party [editor integrations](https://docs.astral.sh/ruff/integrations/) for
-    [VS Code](https://github.com/astral-sh/ruff-vscode) and [more](https://github.com/astral-sh/ruff-lsp)
+    [VS Code](https://github.com/astral-sh/ruff-vscode) and [more](docs/editors/index.md)
```

Comment:
Can we just use the direct link to the editors section? I think we should link the setup section specifically: https://docs.astral.sh/ruff/editors/setup